### DISCUSSION TITLE
Could you please an option of "restic check" and "restic check --read-data-subset=5\%" to backrest?

### DISCUSSION THREAD

**Discussion by ovizii** [garethgeorge/backrest#149] [Ideas]
Seeing that I have almost completely migrated from autorestic and crontab to backrest, I was looking for an option of periodically running those commands to make sure my backups are fine.

**Comment by garethgeorge** [comment]
Tl;dr yes! supporting restic check is something I want to do. 

I’ve been busy with work projects recently, but once some bandwidth frees up I’m aiming to focus on patching a few UI bugs and then get out a major release that adds some new features.

Would it make sense to run restic check as a part of the prune task? I can see this making implementation easy (for me!) but I also view prune as the most likely thing to corrupt a repo as it’s the only time restic deletes data. Or in your mind should it have separate scheduling options?

**Comment by ovizii** [comment]
Great to hear you're working on it.

I'm not a pro, but I must say, previously to using backrest I was running backups, forgets, prunes and checks all independently. I can't remember off the top of my mind what exactly the restic docs recommended timing-wise but of course the more flexibility, the better.
